test/dev-app/views/test/elastic.lp

```diff
@@ -0,0 +1,48 @@
+<?lua local form = require "sailor.form" ?>
```

Comment:
please use 2 spaces for indentation in this file, it will become much better to read